modules/react/core/index.ts

```diff
@@ -31,24 +33,25 @@ export {
   colors,
   gradients,
   depth,
-  spacing,
-  spacingNumbers,
+  space,
+  spaceNumbers,
   type,
   fontFamily,
   monoFontFamily,
   BrandingColor,
   CanvasDepth,
   CanvasDepthValue,
-  CanvasSpacing,
-  CanvasSpacingValue,
```

Comment:
I can understand why we were exporting `CanvasSpacing` (formerly an enum) from `canvas`, but I'm not sure why `CanvasSpacingValue` (union type) was? I moved them to export with the other types.